AffiliatedPackages.md

```diff
@@ -0,0 +1,111 @@
+# PlanetaryPy Affiliated Packages
+
+PlanetaryPy affiliated packages are an important aspect of the 
+PlanetaryPy community.
+
+An affiliated package is a planetary science related Python package that
+is not part of the `planetarypy` core package, and is not managed by the
+project but is a part of the PlanetaryPy Project community. These
+packages demonstrate a commitment to PlanetaryPy’s goals of improving
+reuse, interoperability, and interface standards for Python planetary
+science packages. In many (but not all) cases, affiliated
+packages also follow similar devlopment processes and package
+templates as for the core package.
+
+PlanetaryPy affiliated packages are autonomous projects created and 
+managed by their own Technical Committees (TC), not the PlanetaryPy TC.
+
+These are primarily software projects of various kinds which already
+exist separately.  They benefit from being PlanetaryPy affiliated packages
+in the following ways: 
+
+1. The PlanetaryPy TC helps them adopt or formalize governance and
+	community standards to foster their existance as an open
+	source community.
+2. They gain the ability to easily reach out to the Planetary
+	Software TC for consultation or mentorship on technical
+	or governance issues.
+3. Being recognized as a PlanetaryPy affiliated package lets others
+	know that their project conforms to community best practices
+	which are likely to result in long-term success for their
+	project.
+
+## Becoming an Affiliated Package
+
+If you are a developer of a planetary science package and would like your
+package to become affiliated with the PlanetaryPy Project, please take
+a look at the [instructions for proposing an affiliated package][review-process]. We
+recommend that you also take a look at the [guidelines for reviewing
+affiliated packages][guidelines] since this will give you a sense of whether
+your package is ready for review. Broadly speaking, your package
+should:
+
+* Be potentially useful to planetary scientists. This can mean
+  useful to a specific sub-domain of planetary science, or more broadly useful
+  to a large fraction of planetary science (or beyond, as long as it is also
+  useful for planetary science).
+
+* Be written in a way that is readable and understandable by others. While 
+  not a strict requirement, we also provide coding guidelines that will make 
```

Comment:
because "readable and understandable" can be very subjective, i wonder if this could scare off potential applicants?

Reply:
Yes, both of those are subjective, and that's what the review process is for.  I think this element is fine, I think we want to encourage understandability and readability, and I don't think it will scare folks off.